hadoop-tools/hadoop-archives/src/test/java/org/apache/hadoop/tools/TestHadoopArchives.java

```diff
@@ -804,5 +804,31 @@ public void testCopyToLocal() throws Exception {
       localFs.delete(tmpPath, true);      
     }
   }
-  
+
+  @Test
+  public void testBlockSize() throws Exception {
+    conf.set(HadoopArchives.HAR_BLOCKSIZE_LABEL, "1m");
+
+    final String inputPathStr = inputPath.toUri().getPath();
+    System.out.println("inputPathStr = " + inputPathStr);
```

Comment:
remove it?

Reply:
@pan3793 I have already removed it, please help review it again.